seclea_ai/seclea_ai.py

```diff
@@ -5,6 +5,7 @@
 import copy
 import os
 import threading
+from queue import Queue
```

Comment:
I think that you need to use Queue from either multithreading or multiprocessing builtin library otherwise it's not guaranteed to be reliable in threaded applications.

Reply:
I have used queue from multiprocessing but it was giving error when i was passing transformation object to queue so i have switched with queue.

Reply:
Okay that’s fine then

Reply:
I have refactored, resolving this.